### ISSUE TITLE
Augur user groups don't work with group names

### ISSUE THREAD

**Issue by JamesKunstle** [oss-aspen/8Knot#294]
If group is 'CHAOSS' and augur username is 'sean' the group isn't found in the search bar under 'CHAOSS' but it is under 'sean'

**Comment by JamesKunstle** [comment]
line 258 of pages/index/index_callbacks.py,
user_group_options has a value of 'lower_name' and a label of 'username_groupname'.

**Comment by JamesKunstle** [comment]
in trying to solve this issue, also note that when a user searches for something multiple times the same name may appear multiple times in the searchbar options. maybe run unique on the options before returning?

**Comment by cdolfi** [comment]
@JamesKunstle Is this still an active issue?

**Comment by JamesKunstle** [comment]
@cdolfi Yes I don't think this is done. I'm hoping that by reimplementing oauth and how we handle the user group data the bug disappear. I'll return to this issue later to check on it once oauth is done.

**Comment by MoralCode** [comment]
possibly related #1105
